src/test/kotlin/uk/co/craigbass/pratura/acceptance/BasketSpec.kt

```diff
@@ -73,5 +87,39 @@ class BasketSpec : Spek({
         basketContents().basketValue.shouldEqual("£10.00")
```

Comment:
You should dispatch this to a currency formatter which takes the Money data type and outputs a string. Dependency inject different formatter based on culture of user using system.